tests/concepts/t3330.nim

```diff
@@ -17,17 +17,17 @@ proc add[T](x: var seq[T]; y: openArray[T])
   first type mismatch at position: 1
   required type for x: var seq[T]
   but expression 'k' is of type: Alias
-proc add(result: var string; x: float)
+proc add(x: var string; y: cstring)
```

Comment:
no action needed here but I see this kind of thing break a lot (where ordering is changed), may be good to start sorting sigmatch errors by some reliable criterion (simplest would be raw string sort) to avoid having to edit tests

Reply:
yea it is annoying. But I don't know what the cause it. Is it that I changed some definitions that symbols get a different order when iterating them? Or is it because I introduce a bug into `+%` so that hash values are calculated differently? Or did I fix `+%` and hash values are calculated differently? I don't know. I am worried because I don't understand it.

Reply:
not sure about the other points raised but I'm tracking this specific issue here https://github.com/nim-lang/Nim/issues/13538